src/Kaleidoscope-NumPad.h

```diff
@@ -32,7 +32,14 @@ class NumPad_ : public kaleidoscope::Plugin {
   kaleidoscope::EventHandlerResult afterEachCycle();
 
  private:
-  static byte row, col;
+
+  void cleanupNumlockState(void);
+  void setKeyboardLEDColors(void);
+  bool getNumlockState(void);
+  void syncNumlockState(bool);
+
+  static uint8_t numpad_lock_key_row;
+  static uint8_t numpad_lock_key_col;
   static bool cleanupDone;
```

Comment:
I'm not sure this is a good name anymore (`cleanupDone`, that is), but I don't have a better suggestion either.